codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java

```diff
@@ -225,7 +225,7 @@ public ChannelPromise doneAllocatingPromises() {
                 doneAllocating = true;
                 if (successfulCount == expectedCount) {
                     promise.setSuccess();
-                    return super.setSuccess();
+                    return setSuccess();
```

Comment:
Correct, it was just misleading because it makes it look like you're calling the base class version when you're not.